helm/defectdojo/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: "2.26.4"
 description: A Helm chart for Kubernetes to install DefectDojo
 name: defectdojo
-version: 1.6.88
+version: 1.6.89
 icon: https://www.defectdojo.org/img/favicon.ico
 maintainers:
```

Comment:
The versioning here is managed by the release pipeline. Please do not update it here